14-minimal/root/opt/app-root/etc/scl_enable

```diff
@@ -0,0 +1,3 @@
+# This will make scl collection binaries work out of box.
+unset BASH_ENV PROMPT_COMMAND ENV
+source scl_source enable rh-nodejs${NODEJS_VERSION}
```

Comment:
If this is valid for RHEL8, then rh-nodejs${NODEJS_VERSION} is not correct. rh-nodejs14 is valid for RHSCL - RHEL7 packages.

Reply:
Not valid for RHEL8 at all since we do not have SCLs on RHEL8. Do not think we are planning to have minimal images on RHEL7, so might as well remove the file now.